service/src/main/java/uk/gov/hmcts/reform/fpl/service/RespondentService.java

```diff
@@ -22,29 +22,23 @@
     @Autowired
     private final ObjectMapper mapper = new ObjectMapper();
 
-    public AboutToStartOrSubmitCallbackResponse setMigratedValue(CaseDetails caseDetails) {
+    public AboutToStartOrSubmitCallbackResponse expandRespondentCollection(CaseDetails caseDetails) {
         Map<String, Object> data = caseDetails.getData();
 
-        if (caseDetails.getData().containsKey("respondents1") || !caseDetails.getData().containsKey("respondents")) {
-            data.put("respondentsMigrated", "Yes");
-
-            if (!caseDetails.getData().containsKey("respondents1")) {
-                // Populates first respondent so UI contains expanded Respondent Object.
-                List<Map<String, Object>> populatedRespondent = new ArrayList<>();
-                populatedRespondent.add(ImmutableMap.of(
-                    "id", UUID.randomUUID().toString(),
-                    "value", ImmutableMap.of(
-                        "party", ImmutableMap.of(
-                            // Variable within CCD party structure must be set to expand Collection.
-                            // PartyId is a hidden field so setting a value will not persist to the db
-                            "partyId", UUID.randomUUID().toString()
-                        )
-                    ))
-                );
-                data.put("respondents1", populatedRespondent);
-            }
-        } else {
-            data.put("respondentsMigrated", "No");
+        if (!caseDetails.getData().containsKey("respondents1")) {
```

Comment:
@swalker125 - you modified that code, however by removing the existing test those lines of code are not longer tested in the below test (RespondentServiceTest). Are you happy with the coverage provided by the controller level test?

Reply:
Good spot Adam. At this point we just vaguely ensure that `respondents1` is populated while it does not really ensure that returned value has what we expect it to have. The purpose of the method is to return list structure with respondent that has partyID populated to make CCD render expanded collection.

Since it is 5m effort to write such test I'd suggest adding unit test that checks returned collection has:
- size of 1
- element 1 has party with party ID populated

Reply:
just written the same tests for applicants, shall add a sad and happy test now 👍

Reply:
Just a heads up - this branch is awaiting a refactoring to move away from operating on maps which is being addressed separately. I shall write the test using the current methods and update in future